### DISCUSSION TITLE
Search Bar Widget

### DISCUSSION THREAD

**Discussion by gabrielgaldino500** [matthaigh27/ChatGPT-android-app#12] [Ideas]
I'm not a developer or anything, just a enthusiastic. I think it be extremely useful and would give the app a lot of visibility with you could add a search bar widget on the app to ask questions to ChatGPT directly from the home screen. Is it possible?

**Comment by matthaigh27** [comment]
I don't think this is within the scope of this app. The idea of this app was to be super lightweight and not require giving my API key to someone else. (Also gpt-4 on the phone).
